config/crds/bases/external-secrets.io_clusterexternalsecrets.yaml

```diff
@@ -526,6 +526,28 @@ spec:
                         description: Immutable defines if the final secret will be
                           immutable
                         type: boolean
+                      manifest:
+                        description: |-
+                          Manifest defines a custom Kubernetes resource to create instead of a Secret.
+                          When specified, ExternalSecret will create the resource type defined here
+                          (e.g., ConfigMap, Custom Resource) instead of a Secret.
+                          WARNING: Non-Secret resources are not encrypted at rest. Use with caution.
```

Comment:
idk about this. even secret values need to be configured correctly on cluster side to be encrypted at rest no?